config.go

```diff
@@ -0,0 +1,284 @@
+package cli
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/cheynewallace/tabby"
+	"github.com/urfave/cli/v2"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	configDir  = ".stream-cli"
+	configFile = "config"
+
+	defaultEdgeURL = "https://chat.stream-io-api.com"
+)
+
+func NewRootConfigCmd(config *Config) *cli.Command {
+	return &cli.Command{
+		Name:        "config",
+		Usage:       "Manage app configurations",
+		Description: `Manage app configurations`,
+		Subcommands: []*cli.Command{
+			newConfigCmd(config),
+			removeConfigCmd(config),
+			listConfigsCmd(config),
+			defaultConfigCmd(config),
+		},
+	}
+}
+
+func newConfigCmd(config *Config) *cli.Command {
+	return &cli.Command{
+		Name:        "new",
+		Usage:       "Add a new App configuration",
+		UsageText:   "stream-cli config new",
+		Description: "Add a new App configuration which can be used on further operations",
+
+		Action: func(ctx *cli.Context) error {
+			newConfig := newDefaultConfig()
+			err := survey.Ask(questions(), &newConfig)
+			if err != nil {
+				return err
+			}
+
+			return config.Add(newConfig)
+		},
+	}
+}
+
+func removeConfigCmd(config *Config) *cli.Command {
+	return &cli.Command{
+		Name:        "remove",
+		Usage:       "Remove an App configuration",
+		UsageText:   "stream-cli config remove <app to remove>",
+		Description: "Remove an App configuration. This operation is irrevocable",
+
+		Action: func(ctx *cli.Context) error {
+			if ctx.Args().Len() != 1 {
+				return errors.New("remove command requires 1 argument")
+			}
+			return config.Remove(ctx.Args().First())
+		},
+	}
+}
+
+func listConfigsCmd(config *Config) *cli.Command {
+	return &cli.Command{
+		Name:        "list",
+		Usage:       "List all configurations",
+		UsageText:   "stream-cli config list",
+		Description: "List all app configurations",
+
+		Action: func(_ *cli.Context) error {
+			t := tabby.New()
+			t.AddHeader("", "Name", "Access Key", "Secret Key", "Region")
+
+			for k, v := range config.appsConfig {
+				defaultApp := ""
+				if v.Default {
+					defaultApp = "(default)"
+				}
+
+				secret := fmt.Sprintf("**************%v", v.AccessSecretKey[len(v.AccessSecretKey)-4:])
```

Comment:
What if `AccessSecretKey` has `len == 3`

Reply:
we should add validator for key and secret, let me add this